### ISSUE TITLE
Warning:  Could not validate integrity of download from https://repo.maven.apache.org/maven2/com/microsoft/onnxruntime/xxx

### ISSUE THREAD

**Issue by langchain4j** [microsoft/onnxruntime#16822]
### Describe the issue

Hi, I am getting the following warning when building my project with Maven:

> [INFO] Downloading from central: https://repo.maven.apache.org/maven2/com/microsoft/onnxruntime/onnxruntime/1.15.1/onnxruntime-1.15.1.pom
> Warning:  Could not validate integrity of download from https://repo.maven.apache.org/maven2/com/microsoft/onnxruntime/onnxruntime/1.15.1/onnxruntime-1.15.1.pom
> org.eclipse.aether.transfer.ChecksumFailureException: Checksum validation failed, no checksums available
>     at org.eclipse.aether.internal.impl.AbstractChecksumPolicy.onNoMoreChecksums (AbstractChecksumPolicy.java:64)
>     at org.eclipse.aether.connector.basic.ChecksumValidator.validate (ChecksumValidator.java:107)
>     at org.eclipse.aether.connector.basic.BasicRepositoryConnector$GetTaskRunner.runTask (BasicRepositoryConnector.java:460)
>     at org.eclipse.aether.connector.basic.BasicRepositoryConnector$TaskRunner.run (BasicRepositoryConnector.java:364)
>     at org.eclipse.aether.util.concurrency.RunnableErrorForwarder$1.run (RunnableErrorForwarder.java:75)
>     at org.eclipse.aether.connector.basic.BasicRepositoryConnector$DirectExecutor.execute (BasicRepositoryConnector.java:628)
>     at org.eclipse.aether.connector.basic.BasicRepositoryConnector.get (BasicRepositoryConnector.java:262)
>     at org.eclipse.aether.internal.impl.DefaultArtifactResolver.performDownloads (DefaultArtifactResolver.java:514)
>     at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolve (DefaultArtifactResolver.java:402)
>     at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolveArtifacts (DefaultArtifactResolver.java:229)
>     at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolveArtifact (DefaultArtifactResolver.java:207)
>     at org.apache.maven.repository.internal.DefaultArtifactDescriptorReader.loadPom (DefaultArtifactDescriptorReader.java:240)
>     at org.apache.maven.repository.internal.DefaultArtifactDescriptorReader.readArtifactDescriptor (DefaultArtifactDescriptorReader.java:171)
>     at org.eclipse.aether.internal.impl.collect.DefaultDependencyCollector.resolveCachedArtifactDescriptor (DefaultDependencyCollector.java:538)
>     at org.eclipse.aether.internal.impl.collect.DefaultDependencyCollector.getArtifactDescriptorResult (DefaultDependencyCollector.java:523)
>     at org.eclipse.aether.internal.impl.collect.DefaultDependencyCollector.processDependency (DefaultDependencyCollector.java:410)
>     at org.eclipse.aether.internal.impl.collect.DefaultDependencyCollector.processDependency (DefaultDependencyCollector.java:362)
>     at org.eclipse.aether.internal.impl.collect.DefaultDependencyCollector.process (DefaultDependencyCollector.java:349)
>     at org.eclipse.aether.internal.impl.collect.DefaultDependencyCollector.collectDependencies (DefaultDependencyCollector.java:254)
>     at org.eclipse.aether.internal.impl.DefaultRepositorySystem.collectDependencies (DefaultRepositorySystem.java:284)
>     at org.apache.maven.project.DefaultProjectDependenciesResolver.resolve (DefaultProjectDependenciesResolver.java:170)
>     at org.apache.maven.lifecycle.internal.LifecycleDependencyResolver.getDependencies (LifecycleDependencyResolver.java:266)
>     at org.apache.maven.lifecycle.internal.LifecycleDependencyResolver.resolveProjectDependencies (LifecycleDependencyResolver.java:170)
>     at org.apache.maven.lifecycle.internal.MojoExecutor.ensureDependenciesAreResolved (MojoExecutor.java:401)
>     at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:347)
>     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
>     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:171)
>     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:163)
>     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
>     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
>     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
>     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
>     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:299)
>     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:193)
>     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:106)
>     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:963)
>     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:296)
>     at org.apache.maven.cli.MavenCli.main (MavenCli.java:199)
>     at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
>     at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
>     at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
>     at java.lang.reflect.Method.invoke (Method.java:498)
>     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
>     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
>     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
>     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
> Warning:  Checksum validation failed, no checksums available from central for https://repo.maven.apache.org/maven2/com/microsoft/onnxruntime/onnxruntime/1.15.1/onnxruntime-1.15.1.pom

### To reproduce

Add this to your pom:
```
        <dependency>
            <groupId>com.microsoft.onnxruntime</groupId>
            <artifactId>onnxruntime</artifactId>
            <version>1.15.1</version>
        </dependency>
```

Run
`mvn --batch-mode test`

### Urgency

not urgent

### Platform

Linux

### OS Version

Ubuntu 22.04.2

### ONNX Runtime Installation

Released Package

### ONNX Runtime Version or Commit ID

1.15.1

### ONNX Runtime API

Java

### Architecture

X64

### Execution Provider

Other / Unknown

**Comment by Craigacp** [comment]
The checksum files are there - https://repo.maven.apache.org/maven2/com/microsoft/onnxruntime/onnxruntime/1.15.1/. Do you get similar errors downloading other dependencies?

**Comment by langchain4j** [comment]
No, other deps are good. There are no .md5 and .sha1 files. Maybe this is the problem? Thanks.

**Comment by snnn** [comment]
The checksum files were manually generated by me. Starting from the last release, we use a different workflow for publishing maven packages. And the new system rejects sha1 because it is not secure enough. So, please upgrade your maven.

**Comment by Craigacp** [comment]
We should update the docs to rev the maven version required.

**Comment by jtnord** [comment]
Repository managers may well validate all of the checksums and block if any is incorrect.  This is not a maven version problem it is a checksum generation problem.

**Comment by snnn** [comment]
I thought the SHA256 checksum files can be used as a replacement for SHA1 checksums. At first we used SHA1 checksum, then we migrated our publishing pipeline to a new platform which refuse accepting SHA1 checksums. I will contact their support.
